### PR TITLE
🚀 Release `v4.3.0`

### PR DESCRIPTION
**Pull request by mbercx**

### CODE REVIEW

**Comment by mbercx** [comment]
Hmm, I probably have to first merge [8cca745](https://github.com/aiidateam/aiida-quantumespresso/pull/920/commits/8cca745a36ceeb5f26156e2d93c0379aed0bf0da) before the workflow will activate upon "opening a pull request". Let me play around with it some more.

**Comment by mbercx** [comment]
Blocked until we merge the PR with the GitHub worfklow: https://github.com/aiidateam/aiida-quantumespresso/pull/921

**Comment by mbercx** [comment]
@sphuber if we now consistently sort the commits in the sections proposed in https://github.com/aiidateam/aiida-quantumespresso/pull/921, I would propose that anything added to the following sections should be discussed in the long form message at the top of the changelog:

### ‼️ Breaking changes
### ✨ New features
### 🗑️ Deprecation

Sometimes I wasn't sure when to add a commit to "New features" or "Improvements". I think the distinguishing aspect should be that "New features" are something you want to discuss in the long form statement.

**Comment by mbercx** [comment]
@sphuber two points:

1. I haven't added a lot of code examples here. I'm not sure how to do so without making the `CHANGELOG.md` super long. 
2. For the breaking changes: I had the inclination to explain _why_ the breaking changes are made, but maybe it's best to be succinct. I.e. adopt the approach of the first bullet point and not the second one. The idea is that they need to be able to quickly parse breaking changes, without the clutter of _why_. They can read that in the commits below.

@PNOGillespie @superstar54 can you maybe add a short paragraph on the new XPS features? Thanks a million! 🙏

**Comment by superstar54** [comment]
@mbercx 

> can you maybe add a short paragraph on the new XPS features?

Done. Please feel free to reformat the paragraph if needed.

**Comment by mbercx** [comment]
> Done. Please feel free to reformat the paragraph if needed.

Thanks for the lightning-fast response @superstar54!

**Comment by mkotiuga** [comment]
@mbercx  all good :)

File: CHANGELOG.md
Comment (sphuber) [inline CHANGELOG.md:49]:
```suggestion
### 🐛 Bug fixes
```

File: CHANGELOG.md
Comment (sphuber) [inline CHANGELOG.md:59]: If we use sections for category of changes, I think we can remove the prefix from the commit message that does the same.
```suggestion
* Fix the intersphinx URL of the AiiDA documentation [[08532d8](https://github.com/aiidateam/aiida-quantumespresso/commit/08532d8f69d87bd1853bb2a9010854efbc6bbaee)]
```

File: CHANGELOG.md
Comment (sphuber) [inline CHANGELOG.md:65]:
```suggestion
* Symlink `CRASH` and `data-file.xml` files, and reduce fixture file contents [[0a48533](https://github.com/aiidateam/aiida-quantumespresso/commit/0a48533f40e276e5423428e0ca9745bd0b789121)]
```

File: CHANGELOG.md
Comment (sphuber) [inline CHANGELOG.md:69]:
```suggestion
* Update `pylint` version [[9f4142d](https://github.com/aiidateam/aiida-quantumespresso/commit/9f4142d4713b0d1c32042b3ca35905d725954bf4)]
```

File: CHANGELOG.md
Comment (sphuber) [inline CHANGELOG.md:73]: Should this not be breaking? If people launch their workchains directly, some important parameters may not be set.

File: CHANGELOG.md
Comment (sphuber) [inline CHANGELOG.md:5]:
```suggestion
Release version `4.3.0` comes with a lot of new features, improvements and bug fixes.
Although this is technically a minor release, this new version does come with some _minor_ breaking changes:
```

**Review by sphuber — CHANGES_REQUESTED** [review]
Thanks @mbercx . Just some minor formatting changes. As for the intro, I have to say that I really dislike writing change notes in the royal we. Most projects just use the passive voice for release/change notes and frankly I don't see a reason to change it.

File: CHANGELOG.md
Comment (mbercx) [reply CHANGELOG.md:73]: Fair point, in case they manually specify their inputs they could have e.g. not set the `bands` calculation.

File: CHANGELOG.md
Comment (mbercx) [reply CHANGELOG.md:5]: Haha, I see I got _too_ excited there. 🙃

**Review by sphuber — APPROVED** [review]

**Pull request merged**
